src/lex-keyword.cpp

```diff
@@ -14,16 +17,16 @@
     ('I' == 73) && ('J' == 74) && ('K' == 75) && ('L' == 76) && ('M' == 77) && \
     ('N' == 78) && ('O' == 79) && ('P' == 80) && ('Q' == 81) && ('R' == 82) && \
     ('S' == 83) && ('T' == 84) && ('U' == 85) && ('V' == 86) && ('W' == 87) && \
-    ('X' == 88) && ('Y' == 89) && ('Z' == 90) && ('[' == 91) &&                \
-    ('\\' == 92) && (']' == 93) && ('^' == 94) && ('_' == 95) &&               \
-    ('a' == 97) && ('b' == 98) && ('c' == 99) && ('d' == 100) &&               \
-    ('e' == 101) && ('f' == 102) && ('g' == 103) && ('h' == 104) &&            \
-    ('i' == 105) && ('j' == 106) && ('k' == 107) && ('l' == 108) &&            \
-    ('m' == 109) && ('n' == 110) && ('o' == 111) && ('p' == 112) &&            \
-    ('q' == 113) && ('r' == 114) && ('s' == 115) && ('t' == 116) &&            \
-    ('u' == 117) && ('v' == 118) && ('w' == 119) && ('x' == 120) &&            \
-    ('y' == 121) && ('z' == 122) && ('{' == 123) && ('|' == 124) &&            \
-    ('}' == 125) && ('~' == 126))
+    ('X' == 88) && ('Y' == 89) && ('Z' == 90) &&                               \
```

Comment:
Hmm, these differences shouldn't happen. Are you using the wrong version of clang-format? See docs/CONTRIBUTING.md.

Reply:
yea i was using clang-format 10